index.js

```diff
@@ -16,6 +16,12 @@ var match = String.prototype.match;
 var bigIntValueOf = typeof BigInt === 'function' ? BigInt.prototype.valueOf : null;
 
 var inspectCustom = require('./util.inspect').custom;
+/* eslint-disable no-restricted-properties */
+if (!inspectCustom && typeof Symbol === 'function' && typeof Symbol['for'] === 'function') {
+    inspectCustom = Symbol['for']('nodejs.util.inspect.custom');
```

Comment:
This will mean that in the browser, however, a symbol will be registered and usable. it seems to me that util.inspect's custom symbol should only ever work in node.

Reply:
Also, this logic only helps in node >= 10.12; the dependency is still required for node 6.6-10.11, so i don’t see the value in this added complexity.